tasks.rb

```diff
@@ -1368,11 +1369,20 @@ def set_hpxml_walls(hpxml_file, hpxml)
       wall.solar_absorptance = nil
       wall.color = HPXML::ColorMedium
     end
+  elsif ['base-enclosure-common-walls.xml'].include? hpxml_file
+    hpxml.walls.add(id: 'CommonWallCrawlspace',
```

Comment:
A crawlspace wall is typically a FoundationWall, not (above-grade) Wall.

It'd be good to have have the file test multiple surfaces: adiabatic FoundationWall, adiabatic Wall, and adiabatic FrameFloor. Otherwise I can't tell if your code will work for those other surface types.